### ISSUE TITLE
Consider not rendering to `body`

### ISSUE THREAD

**Issue by thenickcox** [hoffmanc/dnb_pomodoro#2]
I know it's just an example, but this should be a good bootstrap for code that will eventually be in production, right?

It's [dangerous to render to `body`](https://medium.com/@dan_abramov/two-weird-tricks-that-fix-react-7cf9bbdef375) because React doesn't do well when something's added that not under its control, and a lot of third party code manipulates `body`.

**Comment by hoffmanc** [comment]
Yeah I would definitely switch to some grid system, then attach react to the container of that, but I didn't want to add complexity to this particular repository, as I want it to be the simplest possible set of steps to get react running alongside rails with HMR
